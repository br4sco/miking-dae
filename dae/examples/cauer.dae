variables
_uOut_1, _u_R_1, _u_L_50, _u_L_49, _u_L_48, _u_L_47, _u_L_46, _u_L_45, _u_L_44, _u_L_43, _u_L_42, _u_L_41, _u_L_40, _u_L_39, _u_L_38, _u_L_37, _u_L_36, _u_L_35, _u_L_34, _u_L_33, _u_L_32, _u_L_31, _u_L_30, _u_L_29, _u_L_28, _u_L_27, _u_L_26, _u_L_25, _u_L_24, _u_L_23, _u_L_22, _u_L_21, _u_L_20, _u_L_19, _u_L_18, _u_L_17, _u_L_16, _u_L_15, _u_L_14, _u_L_13, _u_L_12, _u_L_11, _u_L_10, _u_L_9, _u_L_8, _u_L_7, _u_L_6, _u_L_5, _u_L_4, _u_L_3, _u_L_2, _u_L_1, _u_C_51, _u_C_50, _u_C_49, _u_C_48, _u_C_47, _u_C_46, _u_C_45, _u_C_44, _u_C_43, _u_C_42, _u_C_41, _u_C_40, _u_C_39, _u_C_38, _u_C_37, _u_C_36, _u_C_35, _u_C_34, _u_C_33, _u_C_32, _u_C_31, _u_C_30, _u_C_29, _u_C_28, _u_C_27, _u_C_26, _u_C_25, _u_C_24, _u_C_23, _u_C_22, _u_C_21, _u_C_20, _u_C_19, _u_C_18, _u_C_17, _u_C_16, _u_C_15, _u_C_14, _u_C_13, _u_C_12, _u_C_11, _u_C_10, _u_C_9, _u_C_8, _u_C_7, _u_C_6, _u_C_5, _u_C_4, _u_C_3, _u_C_2, _u_C_1, _uIn_1, _i_R_1, _i_L_50, _i_L_49, _i_L_48, _i_L_47, _i_L_46, _i_L_45, _i_L_44, _i_L_43, _i_L_42, _i_L_41, _i_L_40, _i_L_39, _i_L_38, _i_L_37, _i_L_36, _i_L_35, _i_L_34, _i_L_33, _i_L_32, _i_L_31, _i_L_30, _i_L_29, _i_L_28, _i_L_27, _i_L_26, _i_L_25, _i_L_24, _i_L_23, _i_L_22, _i_L_21, _i_L_20, _i_L_19, _i_L_18, _i_L_17, _i_L_16, _i_L_15, _i_L_14, _i_L_13, _i_L_12, _i_L_11, _i_L_10, _i_L_9, _i_L_8, _i_L_7, _i_L_6, _i_L_5, _i_L_4, _i_L_3, _i_L_2, _i_L_1, _i_C_51, _i_C_50, _i_C_49, _i_C_48, _i_C_47, _i_C_46, _i_C_45, _i_C_44, _i_C_43, _i_C_42, _i_C_41, _i_C_40, _i_C_39, _i_C_38, _i_C_37, _i_C_36, _i_C_35, _i_C_34, _i_C_33, _i_C_32, _i_C_31, _i_C_30, _i_C_29, _i_C_28, _i_C_27, _i_C_26, _i_C_25, _i_C_24, _i_C_23, _i_C_22, _i_C_21, _i_C_20, _i_C_19, _i_C_18, _i_C_17, _i_C_16, _i_C_15, _i_C_14, _i_C_13, _i_C_12, _i_C_11, _i_C_10, _i_C_9, _i_C_8, _i_C_7, _i_C_6, _i_C_5, _i_C_4, _i_C_3, _i_C_2, _i_C_1, _t_1, _i_1 : Float
init

equations
_t_1' = 1.;
_u_L_50 = 1. * _i_L_50';
_i_C_50 = 1. * _u_C_50';
_u_L_49 = 1. * _i_L_49';
_i_C_49 = 1. * _u_C_49';
_u_L_48 = 1. * _i_L_48';
_i_C_48 = 1. * _u_C_48';
_u_L_47 = 1. * _i_L_47';
_i_C_47 = 1. * _u_C_47';
_u_L_46 = 1. * _i_L_46';
_i_C_46 = 1. * _u_C_46';
_u_L_45 = 1. * _i_L_45';
_i_C_45 = 1. * _u_C_45';
_u_L_44 = 1. * _i_L_44';
_i_C_44 = 1. * _u_C_44';
_u_L_43 = 1. * _i_L_43';
_i_C_43 = 1. * _u_C_43';
_u_L_42 = 1. * _i_L_42';
_i_C_42 = 1. * _u_C_42';
_u_L_41 = 1. * _i_L_41';
_i_C_41 = 1. * _u_C_41';
_u_L_40 = 1. * _i_L_40';
_i_C_40 = 1. * _u_C_40';
_u_L_39 = 1. * _i_L_39';
_i_C_39 = 1. * _u_C_39';
_u_L_38 = 1. * _i_L_38';
_i_C_38 = 1. * _u_C_38';
_u_L_37 = 1. * _i_L_37';
_i_C_37 = 1. * _u_C_37';
_u_L_36 = 1. * _i_L_36';
_i_C_36 = 1. * _u_C_36';
_u_L_35 = 1. * _i_L_35';
_i_C_35 = 1. * _u_C_35';
_u_L_34 = 1. * _i_L_34';
_i_C_34 = 1. * _u_C_34';
_u_L_33 = 1. * _i_L_33';
_i_C_33 = 1. * _u_C_33';
_u_L_32 = 1. * _i_L_32';
_i_C_32 = 1. * _u_C_32';
_u_L_31 = 1. * _i_L_31';
_i_C_31 = 1. * _u_C_31';
_u_L_30 = 1. * _i_L_30';
_i_C_30 = 1. * _u_C_30';
_u_L_29 = 1. * _i_L_29';
_i_C_29 = 1. * _u_C_29';
_u_L_28 = 1. * _i_L_28';
_i_C_28 = 1. * _u_C_28';
_u_L_27 = 1. * _i_L_27';
_i_C_27 = 1. * _u_C_27';
_u_L_26 = 1. * _i_L_26';
_i_C_26 = 1. * _u_C_26';
_u_L_25 = 1. * _i_L_25';
_i_C_25 = 1. * _u_C_25';
_u_L_24 = 1. * _i_L_24';
_i_C_24 = 1. * _u_C_24';
_u_L_23 = 1. * _i_L_23';
_i_C_23 = 1. * _u_C_23';
_u_L_22 = 1. * _i_L_22';
_i_C_22 = 1. * _u_C_22';
_u_L_21 = 1. * _i_L_21';
_i_C_21 = 1. * _u_C_21';
_u_L_20 = 1. * _i_L_20';
_i_C_20 = 1. * _u_C_20';
_u_L_19 = 1. * _i_L_19';
_i_C_19 = 1. * _u_C_19';
_u_L_18 = 1. * _i_L_18';
_i_C_18 = 1. * _u_C_18';
_u_L_17 = 1. * _i_L_17';
_i_C_17 = 1. * _u_C_17';
_u_L_16 = 1. * _i_L_16';
_i_C_16 = 1. * _u_C_16';
_u_L_15 = 1. * _i_L_15';
_i_C_15 = 1. * _u_C_15';
_u_L_14 = 1. * _i_L_14';
_i_C_14 = 1. * _u_C_14';
_u_L_13 = 1. * _i_L_13';
_i_C_13 = 1. * _u_C_13';
_u_L_12 = 1. * _i_L_12';
_i_C_12 = 1. * _u_C_12';
_u_L_11 = 1. * _i_L_11';
_i_C_11 = 1. * _u_C_11';
_u_L_10 = 1. * _i_L_10';
_i_C_10 = 1. * _u_C_10';
_u_L_9 = 1. * _i_L_9';
_i_C_9 = 1. * _u_C_9';
_u_L_8 = 1. * _i_L_8';
_i_C_8 = 1. * _u_C_8';
_u_L_7 = 1. * _i_L_7';
_i_C_7 = 1. * _u_C_7';
_u_L_6 = 1. * _i_L_6';
_i_C_6 = 1. * _u_C_6';
_u_L_5 = 1. * _i_L_5';
_i_C_5 = 1. * _u_C_5';
_u_L_4 = 1. * _i_L_4';
_i_C_4 = 1. * _u_C_4';
_u_L_3 = 1. * _i_L_3';
_i_C_3 = 1. * _u_C_3';
_u_L_2 = 1. * _i_L_2';
_i_C_2 = 1. * _u_C_2';
_u_L_1 = 1. * _i_L_1';
_i_C_1 = 1. * _u_C_1';
_i_C_51 = 1. * _u_C_51';
_u_R_1 = 1. * _i_R_1;
_uIn_1 = 1. / (1. + exp(-.(_t_1)));
_u_L_24 = (-.1. * _u_C_24 - -.1. * _u_C_23);
_u_L_5 = (-.1. * _u_C_5 - -.1. * _u_C_4);
_u_L_25 = (-.1. * _u_C_25 - -.1. * _u_C_24);
_u_L_12 = (-.1. * _u_C_12 - -.1. * _u_C_11);
_u_L_26 = (-.1. * _u_C_26 - -.1. * _u_C_25);
_u_L_2 = (-.1. * _u_C_2 - -.1. * _u_C_1);
_u_L_27 = (-.1. * _u_C_27 - -.1. * _u_C_26);
_u_L_13 = (-.1. * _u_C_13 - -.1. * _u_C_12);
_u_L_28 = (-.1. * _u_C_28 - -.1. * _u_C_27);
_u_L_6 = (-.1. * _u_C_6 - -.1. * _u_C_5);
_u_L_29 = (-.1. * _u_C_29 - -.1. * _u_C_28);
_u_L_14 = (-.1. * _u_C_14 - -.1. * _u_C_13);
_u_L_30 = (-.1. * _u_C_30 - -.1. * _u_C_29);
_u_C_51 = 1. * _uOut_1;
_u_L_31 = (-.1. * _u_C_31 - -.1. * _u_C_30);
_u_L_15 = (-.1. * _u_C_15 - -.1. * _u_C_14);
_u_L_32 = (-.1. * _u_C_32 - -.1. * _u_C_31);
_u_L_7 = (-.1. * _u_C_7 - -.1. * _u_C_6);
_u_L_33 = (-.1. * _u_C_33 - -.1. * _u_C_32);
_u_L_16 = (-.1. * _u_C_16 - -.1. * _u_C_15);
_u_L_34 = (-.1. * _u_C_34 - -.1. * _u_C_33);
_u_L_3 = (-.1. * _u_C_3 - -.1. * _u_C_2);
_u_L_35 = (-.1. * _u_C_35 - -.1. * _u_C_34);
_u_L_17 = (-.1. * _u_C_17 - -.1. * _u_C_16);
_u_L_36 = (-.1. * _u_C_36 - -.1. * _u_C_35);
_u_L_8 = (-.1. * _u_C_8 - -.1. * _u_C_7);
_u_L_37 = (-.1. * _u_C_37 - -.1. * _u_C_36);
_u_L_18 = (-.1. * _u_C_18 - -.1. * _u_C_17);
_u_L_38 = (-.1. * _u_C_38 - -.1. * _u_C_37);
_u_L_1 = (-.1. * _u_C_1 - -.1. * _uIn_1);
_u_L_39 = (-.1. * _u_C_39 - -.1. * _u_C_38);
_u_L_19 = (-.1. * _u_C_19 - -.1. * _u_C_18);
_u_L_40 = (-.1. * _u_C_40 - -.1. * _u_C_39);
_u_L_9 = (-.1. * _u_C_9 - -.1. * _u_C_8);
_u_L_41 = (-.1. * _u_C_41 - -.1. * _u_C_40);
_u_L_20 = (-.1. * _u_C_20 - -.1. * _u_C_19);
_u_L_42 = (-.1. * _u_C_42 - -.1. * _u_C_41);
_u_L_4 = (-.1. * _u_C_4 - -.1. * _u_C_3);
_u_L_43 = (-.1. * _u_C_43 - -.1. * _u_C_42);
_u_L_21 = (-.1. * _u_C_21 - -.1. * _u_C_20);
_u_L_44 = (-.1. * _u_C_44 - -.1. * _u_C_43);
_u_L_10 = (-.1. * _u_C_10 - -.1. * _u_C_9);
_u_L_45 = (-.1. * _u_C_45 - -.1. * _u_C_44);
_u_L_22 = (-.1. * _u_C_22 - -.1. * _u_C_21);
_u_L_46 = (-.1. * _u_C_46 - -.1. * _u_C_45);
_u_R_1 = 1. * _uOut_1;
_u_L_47 = (-.1. * _u_C_47 - -.1. * _u_C_46);
_u_L_23 = (-.1. * _u_C_23 - -.1. * _u_C_22);
_u_L_48 = (-.1. * _u_C_48 - -.1. * _u_C_47);
_u_L_11 = (-.1. * _u_C_11 - -.1. * _u_C_10);
_u_L_49 = (-.1. * _u_C_49 - -.1. * _u_C_48);
_u_C_50 = -.1. * _uOut_1;
_u_L_50 = (1. * _u_C_49 - -.1. * _uOut_1);
0. = (((-.1. * _i_L_50 - -.1. * _i_C_50) - 1. * _i_R_1) - 1. * _i_C_51);
_i_1 = -.1. * _i_L_1;
_i_C_1 = (1. * _i_L_1 - 1. * _i_L_2);
_i_C_2 = (-.1. * _i_L_3 - -.1. * _i_L_2);
_i_C_3 = (-.1. * _i_L_4 - -.1. * _i_L_3);
_i_C_4 = (1. * _i_L_4 - 1. * _i_L_5);
_i_C_5 = (-.1. * _i_L_6 - -.1. * _i_L_5);
_i_C_6 = (-.1. * _i_L_7 - -.1. * _i_L_6);
_i_C_7 = (-.1. * _i_L_8 - -.1. * _i_L_7);
_i_C_8 = (-.1. * _i_L_9 - -.1. * _i_L_8);
_i_C_9 = (-.1. * _i_L_10 - -.1. * _i_L_9);
_i_C_10 = (-.1. * _i_L_11 - -.1. * _i_L_10);
_i_C_11 = (1. * _i_L_11 - 1. * _i_L_12);
_i_C_12 = (-.1. * _i_L_13 - -.1. * _i_L_12);
_i_C_13 = (-.1. * _i_L_14 - -.1. * _i_L_13);
_i_C_14 = (-.1. * _i_L_15 - -.1. * _i_L_14);
_i_C_15 = (-.1. * _i_L_16 - -.1. * _i_L_15);
_i_C_16 = (-.1. * _i_L_17 - -.1. * _i_L_16);
_i_C_17 = (-.1. * _i_L_18 - -.1. * _i_L_17);
_i_C_18 = (-.1. * _i_L_19 - -.1. * _i_L_18);
_i_C_19 = (-.1. * _i_L_20 - -.1. * _i_L_19);
_i_C_20 = (-.1. * _i_L_21 - -.1. * _i_L_20);
_i_C_21 = (-.1. * _i_L_22 - -.1. * _i_L_21);
_i_C_22 = (-.1. * _i_L_23 - -.1. * _i_L_22);
_i_C_23 = (1. * _i_L_23 - 1. * _i_L_24);
_i_C_24 = (-.1. * _i_L_25 - -.1. * _i_L_24);
_i_C_25 = (-.1. * _i_L_26 - -.1. * _i_L_25);
_i_C_26 = (-.1. * _i_L_27 - -.1. * _i_L_26);
_i_C_27 = (-.1. * _i_L_28 - -.1. * _i_L_27);
_i_C_28 = (-.1. * _i_L_29 - -.1. * _i_L_28);
_i_C_29 = (-.1. * _i_L_30 - -.1. * _i_L_29);
_i_C_30 = (-.1. * _i_L_31 - -.1. * _i_L_30);
_i_C_31 = (-.1. * _i_L_32 - -.1. * _i_L_31);
_i_C_32 = (-.1. * _i_L_33 - -.1. * _i_L_32);
_i_C_33 = (-.1. * _i_L_34 - -.1. * _i_L_33);
_i_C_34 = (-.1. * _i_L_35 - -.1. * _i_L_34);
_i_C_35 = (-.1. * _i_L_36 - -.1. * _i_L_35);
_i_C_36 = (-.1. * _i_L_37 - -.1. * _i_L_36);
_i_C_37 = (-.1. * _i_L_38 - -.1. * _i_L_37);
_i_C_38 = (-.1. * _i_L_39 - -.1. * _i_L_38);
_i_C_39 = (-.1. * _i_L_40 - -.1. * _i_L_39);
_i_C_40 = (-.1. * _i_L_41 - -.1. * _i_L_40);
_i_C_41 = (-.1. * _i_L_42 - -.1. * _i_L_41);
_i_C_42 = (-.1. * _i_L_43 - -.1. * _i_L_42);
_i_C_43 = (-.1. * _i_L_44 - -.1. * _i_L_43);
_i_C_44 = (-.1. * _i_L_45 - -.1. * _i_L_44);
_i_C_45 = (-.1. * _i_L_46 - -.1. * _i_L_45);
_i_C_46 = (-.1. * _i_L_47 - -.1. * _i_L_46);
_i_C_47 = (-.1. * _i_L_48 - -.1. * _i_L_47);
_i_C_48 = (-.1. * _i_L_49 - -.1. * _i_L_48);
_i_C_49 = (-.1. * _i_L_50 - -.1. * _i_L_49)
output

{_uOut_1}

