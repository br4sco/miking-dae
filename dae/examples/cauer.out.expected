-5.92148838682e-31
